src/services/ExposureNotificationService/ExposureNotificationService.ts

```diff
@@ -131,10 +131,10 @@ export class ExposureNotificationService {
       await this.exposureNotification.start();
     } catch (error) {
       captureException('Cannot start EN framework', error);
-      this.systemStatus.set(SystemStatus.Unknown);
```

Comment:
This is not necessary anymore as `updateSystemStatus` below will update to proper state.